cli/scorecard/violations.go

```diff
@@ -67,20 +95,40 @@ func addDataFromBucket(config *ScoringConfig, bucket *storage.BucketHandle, obje
 func getViolations(inventory *InventoryConfig, config *ScoringConfig) (*validator.AuditResponse, error) {
 	v := config.validator
 
-	ctx := context.Background()
-	client, err := storage.NewClient(ctx)
-	if err != nil {
-		return nil, err
-	}
+	if inventory.gcsBucket != "" {
+		ctx := context.Background()
+		client, err := storage.NewClient(ctx)
+		if err != nil {
+			return nil, err
+		}
 
-	bucket := client.Bucket(inventory.gcsBucket)
-	for _, objectName := range destinationObjectNames {
-		err = addDataFromBucket(config, bucket, objectName)
+		bucket := client.Bucket(inventory.gcsBucket)
+		it := bucket.Objects(ctx, nil)
```

Comment:
This is something of an interface change which I'd like to consider in a separate PR. In particular, I'm not entirely sure want to switch to scanning all objects in the bucket instead of well-known ones. Could you back this out?

Reply:
Backed out in https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/pull/314/commits/7801912bc8ac86403880f8b5dcaa73c6e1c9e05b

I'm hoping the tool can take multiple resource and iam files at a time because Forseti has `composite_root_resources` field in server config which allows users to scan multiple org/folders/projects at the same time, the CAI exports generated by Forseti have unique file names such as `projects-<project_id>-iam_policy-<timestamp>.dump` and `projects-<project_id>-resource-<timestamp>.dump`
One use case I'm hoping this tool to support would be running the tool against the latest CAI export files in Forseti CAI bucket.
On local file systems it's relatively easy to create sub folders for each node (org, folder, or project) resource and organize CAI export files with expected file names into sub folders, then run a for loop. On GCS it's more difficult as it involves creating additional buckets or some additional code change.

Reply:
Agreed, happy to look at a PR which separately does that. We'll just need to consider whether it should scan *all* the files or only a subset (to capture the latest dump).

Reply:
Ideally only a subset (to capture the latest dump), but the file names do not make it straight forward to extract those. Agreed to revisit this in a separate PR.